### PR TITLE
Support CORS

### PR DESCRIPTION
**Pull request by mark-juwai**
## Summary of changes

- Support CORS

## How to Test

- Run `$ vagrant up`
- Run `$ vagrant ssh -c 'cat /etc/nginx/conf.d/test.conf'`, you can see CORS related config items `Access-Control-*` in the nginx config file.

**Nginx config refers to:**
http://enable-cors.org/server_nginx.html

### CODE REVIEW

File: tests/group_vars/all.yml [hunk 1]
```diff
@@ -30,5 +30,6 @@ dot_env:
 
 server_name: 'test'
 nginx_port: 80
+access_control_allow_origin: '*'
```
Comment (tjoelsson) [inline tests/group_vars/all.yml]: ? sort alphabetically

File: templates/nginx.conf.j2 [hunk 2]
```diff
@@ -36,6 +36,9 @@ server {
     }
 
     location ~ \.php {
+        {% if access_control_allow_origin is defined %}
+        add_header 'Access-Control-Allow-Origin' '{{ access_control_allow_origin }}';
```
Comment (tjoelsson) [inline templates/nginx.conf.j2]: Is this the only setting required?

Comment (mark-juwai) [reply templates/nginx.conf.j2]: Yes, as far as I know.

File: tests/group_vars/all.yml (hunk 1 above)
Comment (mark-juwai) [reply tests/group_vars/all.yml]: OK

File: templates/nginx.conf.j2 (hunk 2 above)
Comment (tjoelsson) [reply templates/nginx.conf.j2]: http://enable-cors.org/server_nginx.html

**Comment by mark-juwai** [comment]
@tjoelsson Comments addressed, please check.

**Comment by tjoelsson** [comment]
Please document variables in readme

**Comment by mark-juwai** [comment]
README updated. @tjoelsson

**Review by tjoelsson — APPROVED** [review]

**Comment by upbeta01** [comment]
:+1:

**Comment by mark-juwai** [comment]
Thank you. @tjoelsson @upbeta01

**Pull request merged**